testing_config.py

```diff
@@ -58,14 +61,12 @@ def setUpOurTestbed():
   ourTestbed.init_app_identity_stub()
   ourTestbed.init_blobstore_stub()
   ourTestbed.init_capability_stub()
-  ourTestbed.init_datastore_v3_stub()
   ourTestbed.init_files_stub()
   ourTestbed.init_logservice_stub()
   ourTestbed.init_mail_stub()
   ourTestbed.init_search_stub()
   ourTestbed.init_urlfetch_stub()
   ourTestbed.init_user_stub()
-  ourTestbed.init_memcache_stub()
 
```

Comment:
We do not require these two stubs. Hence, removed.

Reply:
Great!